elastic-mapping-updater-install/pom.xml

```diff
@@ -0,0 +1,89 @@
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.cafdataprocessing.elastic</groupId>
+        <artifactId>elastic-mapping-updater-aggregator</artifactId>
+        <version>2.3.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>elastic-mapping-updater-install</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.cafdataprocessing.elastic</groupId>
+            <artifactId>elastic-mapping-updater-cli</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+    </dependencies>
+        
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.3.0</version>
+                <executions>
+                    <execution>
+                        <id>create-elastic-mapping-updater-package</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}</outputDirectory>
+                            <appendAssemblyId>false</appendAssemblyId>
+                            <descriptors>
+                                <descriptor>src/main/resources/elastic-mapping-updater.xml</descriptor>
+                            </descriptors>
+                            <formats>
+                                <format>tar.gz</format>
+                            </formats>
+                            <tarLongFileMode>gnu</tarLongFileMode>
+                            <finalName>elastic-mapping-updater-${project.version}</finalName>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.3.0</version>
+                <extensions>true</extensions>
+                <executions>
+                    <execution>
+                        <id>attach-artifacts</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>attach-artifact</goal>
+                        </goals>
+                        <configuration>
+                            <artifacts>
+                                <artifact>
+                                    <file>elastic-mapping-updater-${project.version}.tar.gz</file>
+                                    <type>tar.gz</type>
+                                </artifact>
+                            </artifacts>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
I would have expected the assembly plugin to do this so I can't think why this plugin would be needed.  Did you find that it was?